scripts/house_contacts.py

```diff
@@ -1,22 +1,24 @@
 #!/usr/bin/env python
 
-# Update current congressmen's mailing address from clerk.house.gov.
+# Update current House Member's contact info from clerk XML feed
 
-import lxml.html, io
+import requests
+import lxml
 import re
 from datetime import datetime
-import utils
-from utils import download, load_data, save_data, parse_date
+
+from utils import load_data, save_data, parse_date
 
 def run():
 	today = datetime.now().date()
 
-	# default to not caching
-	cache = utils.flags().get('cache', False)
-	force = not cache
-
 	y = load_data("legislators-current.yaml")
 
+	# TODO use download util?
+	xml = requests.get("http://clerk.house.gov/xml/lists/MemberData.xml")
```

Comment:
no HTTPS version of the clerk URL works at the moment unfortunately.  I'll look into that though.